packages/shared-ux/page/analytics_no_data/src/page.component.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { KibanaNoDataPage } from '@kbn/shared-ux-components';
+
+export interface Props {
+  kibanaGuideDocLink: string;
+  onDataViewCreated: (dataView: unknown) => void;
+}
+
+const solution = i18n.translate('sharedUX.noDataConfig.analytics', {
+  defaultMessage: 'Analytics',
+});
+
+const pageTitle = i18n.translate('sharedUX.noDataConfig.analyticsPageTitle', {
+  defaultMessage: 'Welcome to Analytics!',
+});
+
+const addIntegrationsTitle = i18n.translate('sharedUX.noDataConfig.addIntegrationsTitle', {
+  defaultMessage: 'Add integrations',
+});
+
+const addIntegrationsDescription = i18n.translate(
+  'sharedUX.noDataConfig.addIntegrationsDescription',
+  {
+    defaultMessage: 'Use Elastic Agent to collect data and build out Analytics solutions.',
+  }
+);
+
+export const AnalyticsNoDataPage = ({ kibanaGuideDocLink, onDataViewCreated }: Props) => {
```

Comment:
Should the file itself be named something other than `page.tsx`? Is this the direction we want to go in, ie. having descriptive names for packages and then basics for the component itself?

Reply:
Yeah, probably... just a factor of me moving fast.  I'll correct it.